src/Explorer/Explorer.ts

```diff
@@ -1742,6 +1744,10 @@ export default class Explorer {
       return;
     }
 
+    if (event.data.data.type === MessageTypes.HideConnectScreen) {

```

Comment:
Why is this needed? If you start explorer with "platform=Portal" in the query string shouldn't it work?

Reply:
changed to "platform=Portal" and removed this.